cmd/cloud/deployment.go

```diff
@@ -118,6 +119,7 @@ func newDeploymentCreateCmd() *cobra.Command {
 	cmd.Flags().StringVarP(&description, "description", "d", "", "Description of the Deployment. If the description contains a space, specify the entire description in quotes \"\"")
 	cmd.Flags().StringVarP(&clusterID, "cluster-id", "c", "", "Cluster to create the Deployment in")
 	cmd.Flags().StringVarP(&runtimeVersion, "runtime-version", "v", "", "Runtime version for the Deployment")
+	cmd.Flags().StringVarP(&dagDeploy, "dag-deploy", "", "disable", "To enable or disable dag deploy for the Deployment")
```

Comment:
why not use a boolean flag instead of a string?

Reply:
To keep it consistent with update command.

Keeping it as boolean in both places means having a default value of true or false...and there are scenarios in the update command where we dont need to update dag deploy